### DISCUSSION TITLE
Is there a way to add multiple arrow keys in the shortcut?

### DISCUSSION THREAD

**Discussion by UmairKhalid98** [rxhanson/Rectangle#719] [Q&A] [answered]
I am trying to add two arrow keys + ctrl + option but it only registers one arrow key

**Comment by rxhanson** [comment, marked as answer]
Unfortunately, this is a macOS limitation. Rectangle uses a macOS API to register keyboard shortcuts, and they are limited to 1 key and any number of modifier keys.

**Reply by UmairKhalid98** [reply]
Ah I see, thanks!
